src/functions/supsub.js

```diff
@@ -231,6 +241,10 @@ defineFunctionBuilders({
             if (base && base.type === "op" && base.limits &&
                 (options.style === Style.DISPLAY || base.alwaysHandleSupSub)) {
                 nodeType = "munder";
+            } else if (base && base.type === "operatorname" &&
+                base.alwaysHandleSupSub &&
+                (options.style === Style.DISPLAY || base.limits)) {
```

Comment:
nit: can you make the order of this `||` match the one line line 234?

Reply:
Is there a case where `base.alwaysHandleSupSub` and `base.limits` is falsy?

Reply:
> Is there a case where `base.alwaysHandleSupSub` and `base.limits` is falsy?

Yes. `\lim\nolimits_x y`